modules/migration-combining-must-gather.adoc

```diff
@@ -18,3 +18,10 @@ The only other variables that you can specify this way are the following:
 
 * `logs_since`, with a default value of `72h`
 * `request_timeout`, with a default value of `0s`
+
```

Comment:
If the `DataProtectionApplication` (DPA) custom resource (CR)...

I think this resource justifies this: https://github.com/openshift/oadp-operator/blob/master/docs/examples/stateless.md